README.md

```diff
@@ -3,7 +3,7 @@
 See individual README files for more information.
 
 * [maxscale-cdc-adapter](maxscale-cdc-adapter/README.md)
-* [maxscale-kafka-adapter](maxscale-kafka-adapter/README.md)
+* [maxscale-kafka-adapter](maxscale-kafka-adapter/README.md) (deprecated and replaced by kafka-avro-adapter)
```

Comment:
The two adapter don't do the same thing so calling it a replacement is not accurate. Perhaps just mention that this one is deprecated?

Reply:
thanks, will change it